chapter-3-basics-II.md

```diff
@@ -72,7 +65,7 @@ Under the hood, however, the parameters can do even more: Instances of the
 process and product will be provided to the Populate method and can thus 
 get lots of information. For example, order numbers can be read from the recipe or the ProductType can be read from the process. 
 
-Now add the color as property to the `ColorizingParameters`. Since the property is automatically set using the product information, it doesn't need to be displayed in the UI. 
+Now add the color as property to the `ColorizingParameters` found in the folder `Activities`. Since the property is automatically set using the product information, it doesn't need to be displayed in the UI. 
 
 When populating parameters, the current object always represents the parameters in the workplan, while `instance` (the parameter of the populate method) holds the parameters passed by the activity. The resource gets them during the `OnActivityStarted` method. 
 Now you have to set the color of `instance` to the color of the product. In order to get the ProductType, cast the process to a `ProductionProcess` and get the ProductType from the ProductInstance.
```

Comment:
Maybe it's just my lack of general experience with big projects like this, but I find this text a little bit hard to understand (I dont really understand it myself 😪). I would suggest some kind of flow chart which shows what happens when someone starts production. How do the capabilities interact with the cells and the process engine task ? activity ? step ? recipe?

Reply:
see https://github.com/MORYX-Industry/application-developer-training/pull/3/files